src/libstore/store-api.cc

```diff
@@ -778,24 +760,23 @@ bool ValidPathInfo::isContentAddressed(const Store & store) const
         printError("warning: path '%s' claims to be content-addressed but isn't", store.printStorePath(path));
     };
 
-    if (hasPrefix(ca, "text:")) {
-        Hash hash(std::string(ca, 5));
-        if (store.makeTextPath(path.name(), hash, references) == path)
+    if (! ca) {}
```

Comment:
this should jut return false here to make that explicit

Reply:
I cleaned this one up a good bit :)